### PR TITLE
fixed dependency issue

### PR DESCRIPTION
**Pull request by jainaman224**
Added missing dependency files and made file alphabetically sorted.

### CODE REVIEW

**Comment by rfleschenberg** [comment]
Thanks! I think this needs some adjustments. I'll look into it as soon as I have some time.

**Comment by Aiky30** [comment]
I think virtualenv should be removed from this list too.

**Comment by jainaman224** [comment]
I just did pip freeze to file. Should I add only required packages?

**Comment by Aiky30** [comment]
@jainaman224 I was under the impression that the file is a list of a setup that is proven to be stable. I would personally only have a list of the packages that are 100% required rather than a list of someones current setup. @jrief will define how this file should be managed.

**Comment by rfleschenberg** [comment]
We should not add this line: https://github.com/awesto/django-shop/pull/714/files#diff-b80ca39e5ba09203c741fcc8e103bd16R31

We should also not add anything that is included by other files in the `requirements` directory, for example here: https://github.com/awesto/django-shop/blob/master/requirements/django110.txt

File: requirements/common.txt [hunk 1]
```diff
@@ -1,11 +1,19 @@
+beautifulsoup4==4.6.0
+bs4==0.0.1
```
Comment (rfleschenberg) [inline requirements/common.txt]: Are these two packages required to run the shop, or are they only required to run the tests? Beautifulsoup is also included in https://github.com/awesto/django-shop/blob/master/requirements/test.txt

File: requirements/common.txt [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+certifi==2017.11.5
+chardet==3.0.4
+defusedxml==0.5.0
+Django==1.10.8
```
Comment (rfleschenberg) [inline requirements/common.txt]: Django is included in the more specific files (e.g. `django110.txt`) and should not be included here.

File: requirements/common.txt [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
 django-admin-sortable2==0.6.19
 django-allauth==0.23.0
+django-angular==2.0.2
```
Comment (rfleschenberg) [inline requirements/common.txt]: Same as for Django.

File: requirements/common.txt [hunk 4]
```diff
@@ -19,33 +27,37 @@ django-redis-sessions==0.6.1
 django-rest-auth==0.9.2
 django-sass-processor==0.5.5
 django-sekizai==0.10.0
+django-select2==6.0.2
+-e git+https://github.com/awesto/django-shop@6fdd9428b10241d72ff612365e79c5f649952795#egg=django_shop
```
Comment (rfleschenberg) [inline requirements/common.txt]: Delete this line.

**Comment by rfleschenberg** [comment]
To clarify my earlier comment: Yes, please add only packages that are required.

**Pull request closed without merging**